### PR TITLE
an/invoices

### PR DESCRIPTION
**Pull request by aNorthSense**
1 page focused generalized invoice config

### CODE REVIEW

**Comment by damonbogich** [comment]
@aNorthSense It looks like there's a conflict with the readme that needs to be resolved

File: invoices/config.json
Comment (damonbogich) [inline invoices/config.json:1]: I think primary should be 'invoice'

**Review by damonbogich — COMMENTED** [review]
All looks good except I think the config.json's primary value should be invoice instead of invoices

**Comment by aNorthSense** [comment]
@damonbogich I made the adjustment and also swapped in an anonymized version of the files.

**Pull request merged**